package.json

```diff
@@ -1,78 +1,52 @@
 {
-  "name": "@ngui/auto-complete",
-  "version": "2.0.0",
-  "description": "Angular Input Autocomplete",
-  "license": "MIT",
-  "main": "dist/auto-complete.umd.js",
-  "module": "dist/index.js",
-  "typings": "dist/index.d.ts",
+  "name": "auto-complete",
```

Comment:
```suggestion
  "name": "@ngui/auto-complete",
```
Package name should be still `@ngui/auto-complete`

Reply:
It should be fixed. But refer - `auto-complate/projects/auto-complete/package.json` file.